packages/mobile/src/screens/notifications-screen/Reaction/ReactionList.tsx

```diff
@@ -60,37 +73,37 @@ export const ReactionList = (props: ReactionListProps) => {
     []
   )
 
+  const handlePanResponderRelease = useCallback(() => {
+    onChange(interactingRef.current)
+    interactingRef.current = null
+    setInteracting(null)
+    setGesturesDisabled?.(false)
+  }, [onChange])
+
   const panResponder = useRef(
     PanResponder.create({
-      onMoveShouldSetPanResponder: () => true,
       onPanResponderGrant: (e, gestureState) => {
         setGesturesDisabled?.(true)
         handleGesture(e, gestureState)
       },
-      onPanResponderMove: handleGesture,
-      onPanResponderRelease: () => {
-        onChange(interactingRef.current)
-        interactingRef.current = null
-        setInteracting(null)
-        setGesturesDisabled?.(false)
+      onPanResponderMove: (...args) => {
+        handleGesture(...args)
```

Comment:
just curious, howcome we're spreading these args now?

Reply:
oh nice catch, dont need too, just an artifact of calling a console.log in there